algorithms/kernel/svm/svm_train_boser_impl.i

```diff
@@ -486,19 +478,26 @@ SVMTrainTask<algorithmFPType, ParameterType, cpu>::~SVMTrainTask()
  * \brief Initialize the intermediate data used in SVM training
  *
  * \param[in] C     Upper bound in constraints of the quadratic optimization problem
+*  \param[in] wTable        Pointer to numeric table that contains weights
+*  \param[in] yTable        Pointer to numeric table that contains class labels
  */
 template <typename algorithmFPType, typename ParameterType, CpuType cpu>
-services::Status SVMTrainTask<algorithmFPType, ParameterType, cpu>::init(algorithmFPType C, const NumericTablePtr & wTable)
+services::Status SVMTrainTask<algorithmFPType, ParameterType, cpu>::init(algorithmFPType C, const NumericTablePtr & wTable, NumericTable & yTable)
```

Comment:
`ReadColumns<algorithmFPType, cpu> mtY(yTable, 0, 0, _nVectors);`
why not const reff for `yTable`

Reply:
I will correct in the next PR.